src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+generally very fast. That said, we still want to include initial scroll latencies in our new metric
+for various reasons. First, scrolling is fast only because it has been optimized so much, because it
+is so important. But there are still ways for a website to bypass some of the performance gains that
+the browser offers. The most common one in Chrome is to force scrolling to happen on the main
+thread. So our metric should be able to say when this happens and causes poor scrolling performance
+for users. Second, scrolling is just too important to ignore. We worry that if we exclude scrolling
+then we’ll have a big blindspot, and scrolling performance could decrease over time without web
+developers properly noticing.
+
+There are several events that are dispatched when a user scrolls, such as `touchstart`, `touchmove`,
+and `scroll`. Except for the scroll event, this is largely dependent on the device used for
+scrolling: touch events are dispatched when scrolling with the finger on mobile devices, while wheel
+events occur when scrolling with a mouse wheel. The scroll events are fired after initial scrolling
+has completed. And in general, no DOM event blocks scrolling, unless the website uses [non-passive
+event listeners](/uses-passive-event-listeners). So we think of scrolling as decoupled from DOM
+Events altogether. What we want to measure is the time from when the user moves enough to produce a
+scroll gesture until the first frame that shows that scrolling happened.
+
+#### How to define the latency of an interaction?
+
+As we noted above, interactions that have a "down" and "up" component need to be considered
+separately in order to avoid attributing the time the user spent holding their finger down.
+
+For these types of interactions, we’d like the latency to involve the durations of all events
+associated with them. Since event durations for each "down" and "up" part of the interaction can
+overlap, the simplest definition of interaction latency that achieves this is the maximum duration
+of any event associated with it. Referring back to the keyboard diagram from earlier, this would be
+the `keydown` duration, as it is longer than the `keyup`:
+
+{% Img src="image/kns0INkO77RkiEStzHWYrugyWj32/WFPm4W86CqhFsNc1UZuW.jpeg", alt="Keyboard interaction
+with maximum duration highlighted", width="800", height="311" %}
+
+The `keydown` and `keyup` durations may overlap as well. This may happen for instance when the frame
+presented for both events is the same, as in the following diagram:
+
+{% Img src="image/kns0INkO77RkiEStzHWYrugyWj32/e7htYAZb44AW4UeplBwJ.jpeg", alt="Keyboard interaction
+where press and release occur in the same frame", width="800", height="311" %}
+
+There's are pros and cons to this approach of using the maximum, and we're interested in [hearing
+your feedback](#feedback):
+
+* **Con**: It does not capture the full wait time of the user. For instance, it will capture the
```

Comment:
I feel this Con is so much less important than the Pro's.  I hope I am not just biased here, but I'm trying to evaluate all of the considerations for a metric and evaluating the various tradeoffs.  Here is how I would put it:

1: Represents what users actually see/feel
2. Is not affected by User behaviour (like physically long interactions with device)
3. Consistent with other metric definitions (simplicity)
4. In cases of interactions with multiple components which are each painful, captures cases with most overall user pain.


Regarding (1), I would say that using `max` has one more pro that isn't listed here:
`Whenever there is a frame painted between the components of one interaction, it does not feel representative of UX to consider any time longer than any one of the parts.  (And, if there is only one frame, max already covers the full duration for both.)`

Regarding (4), it can be conflicting with (1) and (3).

Finally, (4) feels conflicting with the overall aggregation/normalization strategy, to me.

Reply:
These seem like a rewording of the existing text, except for (1) and it's not clear to me that (1) is particular to max. I did move the con to last, but other than that kept this.